core/script.runtime/src/kotlin/script/extensions/samWithReceiver.kt

```diff
@@ -16,6 +16,11 @@
 
 package kotlin.script.extensions
 
+// discuss
+
+// Is this an appropriate place to put this class?
```

Comment:
I would prefer if this annotation was removed completely and [the same mechanism](https://kotlinlang.org/docs/reference/compiler-plugins.html#how-to-use-all-open-plugin) as in allopen and noarg plugins was used: a configuration parameter that specifies which annotations enable the SamWithReceiver behavior